packages/evolution-frontend/src/components/inputs/InputMapFindPlace.tsx

```diff
@@ -409,7 +409,7 @@ export class InputMapFindPlace<CustomSurvey, CustomHousehold, CustomHome, Custom
                     readOnly={true}
                     ref={this.props.inputRef}
                 />
-                {this.props.widgetConfig.refreshGeocodingLabel && (
+                {this.props.widgetConfig.refreshGeocodingLabel && this.props.widgetConfig.showSearchPlaceButton !== false && (
```

Comment:
showSearchPlaceButton can be a ParsingFunction, I'd expect to see a call to `surveyHelper.parseBoolean` somewhere